pkg/controller/masteruserrecord/sync_test.go

```diff
@@ -56,7 +56,7 @@ func TestSynchronizeSpec(t *testing.T) {
 
 	uatest.AssertThatUserAccount(t, "john", memberClient).
 		Exists().
-		HasSpec(mur.Spec.UserAccounts[0].Spec)
+		MatchEmbeddedSpec(mur.Spec.UserAccounts[0].Spec)
```

Comment:
Same here as above. We should also check UserID and Disabled in all relevant tests.